packages/p2p/src/components/app.jsx

```diff
@@ -19,6 +20,25 @@ import MyProfile from './my-profile/my-profile.jsx';
 import './app.scss';
 
 const allowed_currency = 'USD';
+
+const TemporaryBarredMessage = () => {
+    const { general_store } = useStores();
+
+    const formatted_date = getFormattedDateString(new Date(convertToMillis(general_store?.user_blocked_until)));
+
+    return (
+        <Toast className='p2p-barred-user' is_open={general_store.is_barred}>
+            <Icon icon='IcAlertWarning' size={16} />
```

Comment:
Please update and use the `<HintBox>` component from `@deriv/components` for this rather than creating a new component.